src/InformationOperation.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Symfony\Flex;
+
+use Composer\DependencyResolver\Operation\SolverOperation;
+use Composer\Package\PackageInterface;
+
+class InformationOperation extends SolverOperation
+{
+    protected $package;
```

Comment:
Does it need to be protected?

Reply:
What do you mean ? "private" would be better ?

Reply:
yes. The rule in Symfony is that we use `private` visibility, unless we have a use case where we need `protected`. This is one of the basis of our BC policy.